references.bib

```diff
@@ -165,18 +167,22 @@ @article{kunen1996moufang
     publisher = {Academic Press}
   }
   @article{deng2016characterizations,
-    title     = {Characterizations Of N(2,2,0) Algebras},
-    author    = {Deng, Fang-an and Chen, Lu and Tuo, ShouHeng and Ren, ShengZhang},
-    journal   = {Algebra},
-    volume    = {2016},
-    year      = {2016},
-    publisher = {Hindawi}
-  }
-  @article{didurik2018some,
-    title   = {Some Properties Of Neumann Quasigroups},
-    author  = {Didurik, Natalia N and Shcherbacov, Victor A},
+    author = {Deng, Fang'an and Chen, Lu and Tuo, Shouheng and Ren, ShengZhang},
+    year = {2016},
+    month = {01},
+    pages = {1-7},
+    title = {Characterizations of N(2,2,0) Algebras},
+    volume = {2016},
+    journal = {Algebra},
+    doi = {10.1155/2016/2752681}
+  }
+  @misc{didurik2018some,
```

Comment:
Surely you can find something actually published instead of a preprint for this material?